### ISSUE TITLE
Feature request: "remember" directory

### ISSUE THREAD

**Issue by DSohl** [JPC-AV/JPC_AV_videoQC#78]
The current default folder upon choosing to select directories is /Users/medialab/github/jpc/JPC_AV_videoQC, and I need to navigate to /Users/medialab/Desktop/JPC/today'sdate to select the video directory. Can the GUI open to the location of the previous file that was check'd or spec'd?

Or, I guess @BleakleyMcD's drag-and-drop idea would bypass the need to re-navigate for every usage.

**Comment by eddycolloton** [comment]
Yup, GUI will now remember the parent directory of the last used directory and open the file dialog at that location. Commit details here: https://github.com/JPC-AV/JPC_AV_videoQC/commit/df12052cebc88e0a2b230d7502001a3a1b51d6fe
